### PR TITLE
corrected compiling issues

### PR DESCRIPTION
**Pull request by Felix10R**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,7 +32,7 @@ ASFLAGS           =
 LDFLAGS           =    -Wl,-pie -specs=$(DEVKITPRO)/libnx/switch.specs -g
 LINKS             =    -limgui -lstbi -ldeko3d -lnx
 
-PREFIX            =    aarch64-none-elf-
+PREFIX            =    $(DEVKITPRO)/devkitA64/bin/aarch64-none-elf-
```
Comment (averne) [inline Makefile:35]:
```suggestion
PREFIX            =    aarch64-none-elf-
```

Comment (averne) [reply Makefile:35]: This should be added to your PATH by the makefile later.

Comment (Felix10R) [reply Makefile:35]: Unfortunately I cannot compile the project without this change (make: aarch64-none-elf-g++: No such file or directory). I can not explain exactly the cause of the problem, but it is resolved by specifying the path. By the way, I'm using macOS 10.15.5.

Comment (averne) [reply Makefile:35]: That's weird, probably a macOS quirk.

**Pull request merged**